### PR TITLE
Updating example to use react-native-share from root project

### PR DESCRIPTION
**Pull request by MateusAndrade**
# Overview

This PRS aims at setting the `Example` project to use the root version of react-native-share, making it easier to test changes. It's based on: https://github.com/react-native-community/datetimepicker.

# Test Plan
Run the example project and call the share sheet on both: android and iOS.

- [x] Build android example app on CI
- [x] Build iOS example app on CI
- [x] Update Docs explaining about example app
- [x] Use this branch on a fresh project without problems - Android
- [x] Use this branch on a fresh project without problems - iOS

# iOS Simulator with Example App
RN PR iOS

# Android Simulator with Example App
RN Share Android

### CODE REVIEW

File: example/ios/example/Info.plist [hunk 1]
```diff
@@ -53,14 +51,6 @@
...
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-	    <string>instagram-stories</string>
-        <string>instagram</string>
-		<string>facebook</string>
-		<string>whatsapp</string>
-		<string>mailto</string>
-	</array>
```
Comment (friederbluemle) [inline example/ios/example/Info.plist:63]: Are you sure you don't need these anymore?

Comment (MateusAndrade) [reply example/ios/example/Info.plist:63]: Actually I need it, thanks for pointing this. 😅

Comment (MateusAndrade) [reply example/ios/example/Info.plist:63]: Done!

**Comment by AbbasTheReckoner** [comment]
Is this version working @MateusAndrade  in Ios as expected along with text?

**Pull request merged**